app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +49,139 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    /**
+     * override method
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector|\Illuminate\View\View*
+     */
+    public function showLoginForm()
+    {
+        $attemptEndTime = session()->get('attempt_end_time');
+        $now = Carbon::now();
+
+        if(isset($attemptEndTime) && $attemptEndTime->diffInMinutes($now)<= 60){
+            $timeLeft = $attemptEndTime->diffInMinutes($now);
```

Comment:
you can use $timeLeft twice no need to create $timeLeft1

Reply:
and totally better to use
            $timeLeft = self::MINUTES_IN_HOUR - $attemptEndTime->diffInMinutes($now);

Reply:
please, never use "magic numbers" in your code